README.md

```diff
@@ -1,73 +1,80 @@
-![](https://img.shields.io/badge/Microverse-blueviolet)
-
-# Project Name
-
-> Description the project.
+## BOOK-FLIGHT-BACKEND
+> BOOK-FLIGHT-backEND to is a RESTful API that allows users to book a flight.
 
 
 ## Built With
 
-- Major languages
-- Frameworks
-- Technologies used
+- REACT
+- REDUX
 
-## Live Demo (if available)
+## Backend Repository
 
-[Live Demo Link](https://livedemo.com)
+ [Pull Requests](https://github.com/ZahraArshia/book-flight-backend/pulls)
+ [Kanban Board](https://github.com/ZahraArshia/book-flight-backend/projects/1)
 
 
 ## Getting Started
 
-**This is an example of how you may give instructions on setting up your project locally.**
-**Modify this file to match your project, remove sections that don't apply. For example: delete the testing section if the currect project doesn't require testing.**
-
-
-To get a local copy up and running follow these simple example steps.
+- Clone [this repository](https://github.com/ZahraArshia/book-flight-frontend.git)
+  ```
+  git clone git@github.com:ZahraArshia/book-flight-frontend.git
+  ```
 
-### Prerequisites
+- Navigate to the created folder.
 
-### Setup
+  ```
+  cd Book-Store
```

Comment:
Please change it to cd book-flight-frontend

Reply:
done